packages/mobile/src/store/storeContext.ts

```diff
@@ -36,5 +37,6 @@ export const storeContext: CommonStoreContext = {
     metadataProgramId: Config.METADATA_PROGRAM_ID
   }),
   // Shim in main, but defined in native-reloaded branch
-  audioPlayer
+  audioPlayer,
+  cognito
```

Comment:
FWIW, the cognito stuff can probably be totally omitted (especially from mobile)

Reply:
ohh good to know, well it's there :D